src/api/app/controllers/webui/request_controller.rb

```diff
@@ -260,7 +260,6 @@ def set_bugowner_request_dialog
   end
 
   def set_bugowner_request
-    required_parameters :project, :user, :group
```

Comment:
routes?

Reply:
Because `group` and `user` are optional and `project` is handle in [check_sanity](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/bs_request_action_set_bugowner.rb#L21) method

Reply:
Hm they have not been optional before and both being optional makes no sense right?